usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateAllRequiredFilePresence.java

```diff
@@ -21,6 +21,8 @@
 import org.mobilitydata.gtfsvalidator.usecase.port.RawFileRepository;
 import org.mobilitydata.gtfsvalidator.usecase.port.ValidationResultRepository;
 
+import java.util.List;
+
 public class ValidateAllRequiredFilePresence {
```

Comment:
The name of this should probably be `ValidateAllRequiredFilesPresence` (the `s` is missing).

Reply:
@fabrice-v we should decide of a common naming convention

Reply:
So, as a rule of thumb I try to avoid 's' postfixes as they tend to lead to sometimes hard to debug typos. I would suggest using explicit naming like `headerCollection`or `headerList` instead of `headers` for example. In this specific case maybe `ValidateRequiredFileAll` would fit? Very open to counter arguments.

Reply:
This is certainly a style issue, so if you have strong feelings against the `s` that's ok. Personally, I find it easier to read the code when it's more concise and more closely reflects conversational language.

So:

```java
for (File f : files)
```

...is more natural to me than:

```java
for (File f : fileList)
```

Also, if you change types of the variable (e.g., from a `List` to a `Map`), then you also have to refactor the variable name, which adds more unnecessary chatter to version control.

But, I don't have extremely strong feelings on this.

Not sure if I get the `s` issue with Class names, though...

Reply:
On linking variable name to the implementation, we'll prefer using Collection. 
As for the class name I was thinking about being consistent.